.github/workflows/update_data.yml

```diff
@@ -36,14 +36,9 @@ jobs:
       run: |
         make files
         today=$(date +'%Y-%m-%d')
+        aws s3 ls s3://$AWS_BUCKET/update_data/pdfs/florida/ > existing_assets_florida
```

Comment:
this  bit checks what's in S3 already, so we don't need to spend ages downloading the PDFs all over again